src/envoy/http/authn/http_filter_factory.cc

```diff
@@ -23,19 +23,24 @@ namespace Envoy {
 namespace Server {
 namespace Configuration {
 
+namespace {
+// The name for the mixer server cluster.
```

Comment:
.. name for istio authentication filter.

Reply:
Fixed.